doc/articles/intro.md

```diff
@@ -1,42 +1,42 @@
+Uno Platform provides a common set of layout and controls, designed to provide the ability to share an important part of an application's code and markup; however, it leaves developers with the ability to retain the native look and feel. At the same time, it provides a way to have a *pixel-perfect* UI and UX being identical on all platforms. Commonly, this look and feel will be found in the navigation, transitions and animations, main pages, and edges of the screen.
 
-While the Uno Platform provides all the UWP APIs, a lot of those APIs are not implemented. It currently provides a small set of basic non-UI parts of the UWP, such as the `Windows.UI.Xaml.Application` class, which provides the ability to have a common application bootstrapping code. 
+While the Uno Platform provides all the UWP APIs, a lot of those APIs are not implemented. It currently provides a small set of basic non-UI parts of the UWP, such as the `Windows.UI.Xaml.Application` class, which provides the ability to have a common application bootstrapping code. Providing full support for all UWP APIs is an ongoing effort from Uno Platform core team and community and is prioritized based on community and enterprise support customer needs. 
 
 ## How does Uno Platform work?
 
-Uno provides a set of APIs that use class and property names compatible with Windows UWP, while allowing those classes to inherit from the primitive layout container of the platform, in the case of the XAML APIs.
+Uno Platform provides a set of APIs that use class and property names compatible with Windows UWP, while allowing those classes to inherit from the primitive layout container of the platform, in the case of the XAML APIs.
 
 For instance, `Windows.UI.Xaml.Controls.StackPanel` directly inherits from a `ViewGroup` on Android and `UIView` on iOS. 
 
 The native layout system for inner elements is then overridden with a XAML-compatible layout system, using the standard XAML Measure and Arrange passes. This means that a StackPanel will use the same layout strategies on all platforms, and will, therefore, look the same on-screen.
 
-On Windows platforms, Uno.UI is not present and the XAML-layout files are left untouched. On Xamarin-classic-compatible platforms, the XAML files are processed at compile time to generate non-conditional code that will be executed as-is at runtime on the device. This means that there is no runtime parsing of XAML, which makes the UI-tree creation particularly efficient.
+On Windows platforms, Uno.UI is not present and the XAML-layout files are left untouched. On Xamarin-classic-compatible platforms (note, not Xamarin.Form), the XAML files are processed at compile time to generate non-conditional code that will be executed as-is at runtime on the device. This means that there is no runtime parsing of XAML, which makes the UI-tree creation particularly efficient.
```

Comment:
Sounds more confusing for me, I would instead add a link to an external page which explain what the difference between "Xamarin-classic" and "Xamarin.Form".

Reply:
Agreed that this is confusing as is. Actually the 'Xamarin-classic-compatible' language seems out of date (pre-dating Wasm??) Proposed alternative below.

```suggestion
On Windows platforms, Uno.UI is not present and the XAML-layout files are left untouched. On non-Windows platforms, the XAML files are processed at compile time by Uno.UI to generate non-conditional code that will be executed as-is at runtime on the device. This means that there is no runtime parsing of XAML, which makes the UI-tree creation particularly efficient.

(Note: Uno also offers [a set of renderers allowing Xamarin.Forms applications to run on WebAssembly](https://platform.uno/xamarin-forms/). In this case, the Xaml parsing is handled by Xamarin.Forms and may occur at runtime depending on the project configuration.)
```